### PR TITLE
Update B2C token retrieval endpoint docs

### PR DESCRIPTION
**Pull request by keesschollaart81**
As some people in de comments mention, the Token endpoint URL is incorrect. Also the token endpoint does not return an `access_token` but an `id_token`, I'm not sure if this is an error in the docs or in the behaviour of the API but right now it's confusing.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@keesschollaart81__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

File: articles/active-directory-b2c/active-directory-b2c-reference-oidc.md [hunk 1]
```diff
@@ -166,11 +166,11 @@ If all that your web app needs to do is execute policies, you can skip the next
 You can redeem the authorization_code that you acquired (by using `response_type=code+id_token`) for a token to the desired resource by sending a `POST` request to the `/token` endpoint. Currently, the only resource that you can request a token for is your app's own backend web API. The convention for requesting a token to yourself is to use your app's client ID as the scope:
 
 ```
-POST fabrikamb2c.onmicrosoft.com/v2.0/oauth2/token?p=b2c_1_sign_in HTTP/1.1
+POST fabrikamb2c.onmicrosoft.com/oauth2/v2.0/token?p=b2c_1_sign_in HTTP/1.1
 Host: https://login.microsoftonline.com
 Content-Type: application/x-www-form-urlencoded
 
-grant_type=authorization_code&client_id=90c0fe63-bcf2-44d5-8fb7-b8bbc0b29dc6&scope=90c0fe63-bcf2-44d5-8fb7-b8bbc0b29dc6 offline_access&code=AwABAAAAvPM1KaPlrEqdFSBzjqfTGBCmLdgfSTLEMPGYuNHSUYBrq...&redirect_uri=urn:ietf:wg:oauth:2.0:oob&client_secret=<your-application-secret>
```
Comment (keesschollaart81) [inline articles/active-directory-b2c/active-directory-b2c-reference-oidc.md]: setting the client_id as scope (as suggested paragraph before) will result in a Bad Request

**Comment by PRmerger** [comment]
@keesschollaart81 : Thanks for your contribution to the Azure documentation! The author, @dstrockis, has been notified to review your proposed change.

**Comment by BryanLa** [comment]
Hey @dstrockis, @swkrish  -  any thoughts on this one? Please drop a sign-off if it looks OK and we will get it merged.

**Comment by dstrockis** [comment]
@keesschollaart81 thanks for taking the time to fix these! I fixed the path ordering yesterday, those should be live now.

The scope=client_id thing should actually be supported by the service.  I'm double checking with them on that. I'm going to leave this PR open until I hear back.

**Comment by dstrockis** [comment]
I confirmed this should be supported by the service.  Please close out this PR.

**Comment by ktoliver** [comment]
#close

**Pull request closed without merging**